ui-spacetimechart/src/lib/types.ts

```diff
@@ -99,7 +99,8 @@ export type LayerType = (typeof LAYERS)[number];
 // PICKING SPECIFIC TYPES:
 export type PickingElement =
   | { type: 'point'; pathId: string; point: Point }
-  | { type: 'segment'; pathId: string; from: Point; to: Point };
+  | { type: 'segment'; pathId: string; from: Point; to: Point }
+  | { type: 'conflict'; conflictIndex: number };
```

Comment:
@jacomyal Was this how you envisioned how picking would work for other kinds of elements? Or should we try to make picking more modular?